src/main/java/by/custom/utilcalculator/domain/tree/HelperTree.java

```diff
@@ -2,68 +2,73 @@
 
 import by.custom.utilcalculator.domain.constants.Command;
 import by.custom.utilcalculator.domain.constants.steps.*;
+import by.custom.utilcalculator.exception.TreeReadingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
+import java.io.*;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
 import java.util.*;
 
 public class HelperTree {
+    private static final ObjectMapper mapper = new ObjectMapper();
+
     public static Map<StepsIndicator, String> fillFieldsToCommandsMap() {
-        Map<StepsIndicator, String> fieldsToCommands = new HashMap<>(14);
+        final Map<StepsIndicator, String> fieldsToCommands = new HashMap<>(14);
```

Comment:
Is it possible to use Command as a value instead of String?
```suggestion
        final Map<StepsIndicator, Command> fieldsToCommands = new HashMap<>(14);
```

Reply:
Commands are better than Strings, you explained me why. 
Replaced all Strings to Commands